### DISCUSSION TITLE
Cost of hosting

### DISCUSSION THREAD

**Discussion by RickCogley** [nytimes/library#306] [General]
Hi, we are a small co looking to use Library. Can anyone share what we can expect to spend or need to spend per month on Heroku or GAE? Appreciate any info! 🙏

**Comment by afischer** [comment]
Hi @RickCogley, apologies for the (very) late reply here, but noticed you hadn't gotten any answers. 

The [demo site](https://nyt-library-demo.herokuapp.com/) uses the "Eco" option from Heroku, which costs $5/month. It works fine for a relatively low-traffic site. Heroku also offers an option for a maximum of $7/mo that will not sleep, so it should make for a generally faster and more responsive site.

**Reply by RickCogley** [reply]
Thanks for the info
